relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -189,6 +219,20 @@ fn extract_universal_tags(event: &Event, config: &TransactionMetricsConfig) -> C
         tags.insert(CommonTag::HttpMethod, http_method);
     }
 
+    if let Some(browser_name) = extract_browser_name(event) {
+        tags.insert(CommonTag::BrowserName, browser_name);
+    }
+
+    if let Some(os_name) = extract_os_name(event) {
+        tags.insert(CommonTag::OsName, os_name);
+    }
+
+    if let Some(user) = event.user.value() {
+        if let Some(geo_country_code) = extract_geo_country_code(user) {
+            tags.insert(CommonTag::GeoCountryCode, geo_country_code);
+        }
```

Comment:
nit: The other two `extract_` functions take the full event as parameter, even though they only need `event.contexts`. So I would change the signature of `extract_geo_country_code()` to take an `&Event` as well.

Reply:
makes sense, done in [3fd3299](https://github.com/getsentry/relay/pull/2108/commits/3fd3299cfac6d03b0791cadcda69a38c6ac0c549)